### PR TITLE
fix: rename to aibrix/v6d and fix workflows

### PR DESCRIPTION
**Pull request by DwyaneShi**
- makes necessary renaming
- fixes format issues of cpp code
  - fixes 678ef43 ("v1/api: support query w/ existing buffer")
  - fixes a1dbac7 ("feat: query interface ext")
- adds pkg write permission for workflows pushing pkgs
- fixes fluid fuse image build procedure
  - fixes 78e236e ("Support to start vineyardd in vineyard fluid fuse.")

Note: vineyard operator CI workflow is not enabled and will only be enabled if needed in the future.

### CODE REVIEW

File: .github/workflows/build-vineyardd-and-wheels-linux.yaml [hunk 1]
```diff
@@ -24,10 +24,13 @@ concurrency:
   group: ${{ github.repository }}-${{ github.event.number || github.head_ref || github.sha }}-${{ github.workflow }}
   cancel-in-progress: true
 
+permissions:
```
Comment (xieus) [inline .github/workflows/build-vineyardd-and-wheels-linux.yaml:27]: hmm, this is beyond renaming :-)

File: .github/workflows/build-vineyardd-and-wheels-linux.yaml [hunk 2]
```diff
@@ -268,19 +262,6 @@ jobs:
           aarch64-linux-gnu-strip ./vineyardctl || true
           upx -9 ./vineyardctl || true
 
-      - name: Upload vineyardctl to tagged release
```
Comment (xieus) [inline .github/workflows/build-vineyardd-and-wheels-linux.yaml:271]: Check this as well.

**Review by xieus — COMMENTED** [review]
@DwyaneShi Few minor comments.

File: .github/workflows/build-vineyardd-and-wheels-linux.yaml (hunk 1 above)
Comment (DwyaneShi) [reply .github/workflows/build-vineyardd-and-wheels-linux.yaml:27]: Renamed the PR accordingly. PTAL.

File: .github/workflows/build-vineyardd-and-wheels-linux.yaml (hunk 2 above)
Comment (DwyaneShi) [reply .github/workflows/build-vineyardd-and-wheels-linux.yaml:271]: I've reverted the previous hack commit, PTAL again.

File: .github/workflows/build-vineyardd-and-wheels-linux.yaml (hunk 1 above)
Comment (Jeffwan) [reply .github/workflows/build-vineyardd-and-wheels-linux.yaml:27]: /lgtm

File: test/concurrent_memcpy_test.cc [hunk 3]
```diff
@@ -50,167 +50,168 @@ void testing_memcpy(const size_t size) {
 }
 
 void test_concurrent_memcpy_n_basic() {
-    std::vector<const void*> src_buffers;
```
Comment (Jeffwan) [inline test/concurrent_memcpy_test.cc:53]: does v6d itself have any format tools? not in the CI? I am a little bit worry about the future rebase if we introduce format changes

**Review by Jeffwan — COMMENTED** [review]
all artifact renaming make sense and looks good to me.  I left a minor comment on the format changes

File: test/concurrent_memcpy_test.cc (hunk 3 above)
Comment (DwyaneShi) [reply test/concurrent_memcpy_test.cc:53]: CI right now only does format check but the repo does have a formatting tool that can carry out potential format changes.

BTW, the reason that our previous commits have format issues is that we don't use the same version of clang-format as v6d CI does.

Comment (Jeffwan) [reply test/concurrent_memcpy_test.cc:53]: Got you, make sense

**Review by Jeffwan — APPROVED** [review]

**Pull request merged**